content/doc/book/pipeline/pipeline-as-code.adoc

```diff
@@ -250,8 +250,8 @@ Jenkins experience increases, organizations inevitably want to move beyond
 simple pipelines and create complex flows specific to their delivery process.
 
 These Jenkins users require a feature that treats complex pipelines as a
-first-class object, and so the plugin:workflow-aggregator[Pipeline plugin]
-was developed .
+first-class object, and so the plugin:workflow-aggregator[Pipeline plugin] 
```

Comment:
is the space at the end of the line needed?
```suggestion
first-class object, and so the plugin:workflow-aggregator[Pipeline plugin]
```

Reply:
I don’t think it’s necessary. thanks!